src/nls/fi/strings.js

```diff
@@ -631,7 +631,6 @@ define({
     "EXPAND_ALL"                    : "Laajenna kaikki",
     "COLLAPSE_CURRENT"              : "Pienennä nykyinen",
     "EXPAND_CURRENT"                : "Laajenna nykyinen",
-    "COLLAPSE_CUSTOM_REGIONS"       : "Pienennä mukautetut alueet",
     "MIN_FOLD_SIZE"                 : "Pienin laskostuksen koko",
```

Comment:
Only strings in the UI are put in nls/lang/strings.js file; other strings could be moved into the extension as some constants. The problem with having string constants here is that the automation team would send these strings to for translation or the community would translate these constants, which would be a loss of resources.